ref/ref.mbti

```diff
@@ -1,13 +1,13 @@
 package moonbitlang/core/ref
 
-alias @moonbitlang/core/num as @num
+alias @moonbitlang/core/math as @math
 
 // Values
 fn ref[T](T) -> Ref[T]
 
 // Types and methods
-fn Ref::decr[T : @num.Num](Ref[T], T) -> Unit
-fn Ref::incr[T : @num.Num](Ref[T], T) -> Unit
+fn Ref::decr[T : @math.FromInt + @math.Subtract](Ref[T], T) -> Unit
+fn Ref::incr[T : @math.FromInt + @math.Add](Ref[T], T) -> Unit
```

Comment:
in general, you want incr to be really efficient,
e.g,
```
let  v = ref (0)
incr(v)
```
people would expect `v` stack allocated

Reply:
Now rebased to include #335